src/create-element.js

```diff
@@ -74,9 +75,11 @@ export function createVNode(type, props, key, ref) {
 		// a _nextDom that has been set to `null`
 		_nextDom: undefined,
 		_component: null,
-		constructor: undefined
+		constructor: undefined,
+		_original: original
```

Comment:
Oh! I forgot to mention this in my previous review - can we add `_original` to the `VNode` interface in `internal.d.ts`?